### PR TITLE
IGVF-865-relax-assoc-disease-audit

### PR DESCRIPTION
**Pull request by iyoungworth**
update-mappings

### CODE REVIEW

File: src/igvfd/tests/test_audit_construct_library.py [hunk 1]
```diff
@@ -6,27 +6,8 @@ def test_audit_construct_library_associated_disease(
     base_construct_library,
     phenotype_term_alzheimers
```
Comment (jenjou) [inline src/igvfd/tests/test_audit_construct_library.py:7]: I think phenotype_term_alzheimers can be dropped from this as it's no longer used in the test code.

**Review by jenjou — CHANGES_REQUESTED** [review]
Minor change requested for the audit test. Looks good otherwise

File: src/igvfd/tests/test_audit_construct_library.py (hunk 1 above)
Comment (iyoungworth) [reply src/igvfd/tests/test_audit_construct_library.py]: Thanks for catching! Removed

**Comment by jenjou** [comment]
Looks OK but there is another conflict in the Mappings file.

**Review by jenjou — APPROVED** [review]

**Pull request merged**